ground/src/main/java/com/google/android/ground/model/geometry/Geometry.kt

```diff
@@ -88,6 +88,7 @@ data class MultiPolygon(val polygons: List<Polygon>) : Geometry {
 /** A sequence of two or more vertices modelling an OCG style line string. */
 @Serializable
 @SerialName("line_string")
+// TODO(#2499): Deprecate usages of LineString
```

Comment:
LineString is needed for the polygon drawing tool to represent unclosed/unfinished polygons. We don't serialize these; they're only transient, but I think they still need a representation in the model.

Reply:
Done